videomaker/styles/light.json

```diff
@@ -0,0 +1,30 @@
+{
+	"version": 2,
+	"title": "Light",
+	"settings": {
+		"color": {
+			"palette": [
+				{
+					"slug": "foreground",
+					"color": "#000000",
+					"name": "Foreground"
+				},
+				{
+					"slug": "background",
+					"color": "#FFFFFF",
+					"name": "Background"
+				},
+				{
+					"slug": "primary",
+					"color": "#000000",
+					"name": "Primary"
+				},
+				{
+					"slug": "tertiary",
+					"color": "var:preset|color|tertiary",
```

Comment:
I don't think colors are inherited. The variable isn't working.

<img width="1308" alt="image" src="https://user-images.githubusercontent.com/1935113/190111096-2778d896-bf08-45fb-8334-16f36d6422ce.png">